.github/workflows/ci.yml

```diff
@@ -92,6 +92,8 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
what is this change for?

Reply:
By default `actions/checkout` uses the shallow `fetch-depth: 1` which is faster as it only checks out the last commit on the branch and nothing else. The deep checkout with unlimited depth (`0`) fetches the full history, enabling comparison to `origin/master`.